examples/src/main/java/com/cloudera/nav/plugin/client/examples/updatedResults/GetAllUpdatedResults.java

```diff
@@ -0,0 +1,32 @@
+package com.cloudera.nav.plugin.client.examples.updatedResults;
+
+import com.cloudera.nav.plugin.client.*;
+
+import java.util.Map;
+
+/**Example for using getAllUpdated() methods to retrieve entities and relations modified since a given set of extractions.
+ *
+ * Created by Nadia.Wallace on 6/4/15.
+ */
+public class GetAllUpdatedResults {
+
+    public static void main(String[] args){
+        // setup the plugin and api client
+        String configFilePath = args[0];
+        PluginConfigurations config = (new PluginConfigurationFactory())
+                .readConfigurations(configFilePath);
+        NavigatorPlugin plugin = new NavigatorPlugin(config);
+        NavApiCient client = new NavApiCient(config);
+
+        UpdatedResults resultsNoMarker = client.getAllUpdated();
```

Comment:
Add a comment / change variable name here to make clear it's the initial bulk extract